### ISSUE TITLE
Crucible not updating the Liquid numbers when Solids are empty

### ISSUE THREAD

**Issue by Seyeght** [BloodWorkXGaming/ExNihiloCreatio#31]
Video evidence:
https://clips.twitch.tv/SuspiciousCleverOwlCopyThis

This happened with Thermal Dynamics pipes and with Extra Utilities 2 Transfer Nodes,

**Comment by Seyeght** [comment]
Make Mike work this one, it's probably his fault anyway. :P

**Comment by SirLyle** [comment]
I tested this in my world some. It appears the crucible isn't updating visually/WAILA, but TheOneProbe does get updated, and the lava is actually coming from the fluiduct's internal buffer.

I've attached some screen shots where I set up your conditions, then drained the stuff remaining in the fluiduct, then it stopped filling, but visually and in WAILA showed more fluid in the crucible.

My guess is that the tank.drain() call isn't being feed up to the crucible telling it to update the client (WAILA compat is @SideOnly(Side.CLIENT)). Haven't dug into TOP, but I bet it talks to the server more.

![2017-09-19_22 33 54](https://user-images.githubusercontent.com/6723308/30624333-c986699e-9d8a-11e7-82f3-433c0690db13.png)
![2017-09-19_22 34 11](https://user-images.githubusercontent.com/6723308/30624334-c9914eae-9d8a-11e7-9c07-c118340684a9.png)
![2017-09-19_22 34 16](https://user-images.githubusercontent.com/6723308/30624335-c99c5c0e-9d8a-11e7-916d-c133d54b2f25.png)
